modules/arm_plugin/src/arm_infer_request.hpp

```diff
@@ -35,8 +35,11 @@ struct ArmInferRequest : public InferenceEngine::IInferRequestInternal {
     using Ptr = std::shared_ptr<ArmInferRequest>;
 
     ArmInferRequest(const InferenceEngine::InputsDataMap&     networkInputs,
-                      const InferenceEngine::OutputsDataMap&    networkOutputs,
-                      const std::shared_ptr<ExecutableNetwork>& executableNetwork);
```

Comment:
why have you removed this ctor? just keep as is

Reply:
Constructor retained. That's just tabulation update